pyproject.toml

```diff
@@ -90,13 +93,13 @@ markers = ["notebooks: run notebooks", "bnotebooks: run benchmark notebooks"]
 legacy_tox_ini = """
     [tox]
     requires = tox>=4
-    env_list = format, lint, notebooks, py{38,39,310}
+    env_list = format, lint, notebooks, py{38,39,310,311}
 
     [testenv]
     description = run unit tests
     extras = ci
     commands =
-            pytest tests -m "not notebooks"
+            pytest tests -m "not notebooks" -m "not bnotebooks"
```

Comment:
What does this line do? benchmark notebooks?

Reply:
Yes, so with #583 we now have seperate pytest markers for the benchmark notebooks.